# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/public/fribidi/pkg-config-include-patch.patch
# ROCK Linux is Copyright (C) 1998 - 2006 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

diff -u fribidi-0.10.4/fribidi.pc.in fribidi-0.10.4-fixed/fribidi.pc.in
--- fribidi-0.10.4/fribidi.pc.in	2002-01-03 16:31:24.000000000 +0100
+++ fribidi-0.10.4-fixed/fribidi.pc.in	2003-05-18 22:29:08.000000000 +0200
@@ -7,4 +7,4 @@
 Description: Unicode BiDirectional algorithm library
 Version: @VERSION@
 Libs: -L${libdir} -lfribidi
-Cflags: -I${includedir}/fribidi
+Cflags: -I${includedir} -I${includedir}/fribidi
Common subdirectories: fribidi-0.10.4/tests and fribidi-0.10.4-fixed/tests
Common subdirectories: fribidi-0.10.4/unidata and fribidi-0.10.4-fixed/unidata
